src/activities/Elsa.Activities.SQL/Activities/SqlCommand/ExecuteSqlServerCommand.cs

```diff
@@ -0,0 +1,60 @@
+using Elsa.Services;
+using Elsa.Attributes;
+using Elsa.Services.Models;
+using Elsa.ActivityResults;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Activities.Sql.Factory;
+
+namespace Elsa.Activities.Sql.Activities
+{
+    /// <summary>
+    /// Execute an SQL query on given database using connection string
+    /// </summary>
+    [Trigger(
+        Category = "SQL Server",
+        DisplayName = "Execute Sql Command",
```

Comment:
For Display Name, a proper English name is appropriate: "Execute **SQL** Command" (In English, we should use call-caps for the "SQL" acronym, as opposed in C# where we use Pascal-casing for acronyms that take 3 characters or more).